### ISSUE TITLE
Linux version in 'uanme -a' always show as firstly download image

### ISSUE THREAD

**Issue by wangdong268** [docker/docker#16299]
Steps:
1. Download fedora 22 image use 'docker pull fedora:22'
2. Create a container with F22 image and use 'uname -a' to check linux version. It will show linux version is f22.

3. Download fedora 21 image use 'docker pull fedora:21'
4. Create F21 container and use 'uname -a', result as below:
```
[wangdong@dhcp-136-140 tools-automation]$ sudo docker run -it --name dowang_test21 fedora:21 /bin/bash
bash-4.3# uname -a
Linux 5851876fbb32 4.1.6-200.fc22.x86_64 #1 SMP Mon Aug 17 19:54:31 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
bash-4.3# dnf
bash: dnf: command not found
bash-4.3# exit
```

Why my container use a fedora 21 image, but it show f22 in uname -a .

**Comment by wangdong268** [comment]
Update:

**docker version:**  ```Docker version 1.8.1, build d12ea79```
**docker info:**
```
Containers: 1
Images: 5
Storage Driver: devicemapper
 Pool Name: docker-253:1-2365047-pool
 Pool Blocksize: 65.54 kB
 Backing Filesystem: extfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 2.879 GB
 Data Space Total: 107.4 GB
 Data Space Available: 15.52 GB
 Metadata Space Used: 2.195 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.145 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.93 (2015-01-30)
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 4.1.6-200.fc22.x86_64
Operating System: Fedora 22 (Twenty Two)
CPUs: 4
Total Memory: 7.512 GiB
Name: dhcp-136-140.nay.redhat.com
ID: TFYK:PQMG:X6EP:4KBA:XTMB:RHWY:OJGH:TSUX:NTWW:7YQW:LFEP:FBDW
Username: wangdong268
Registry: https://index.docker.io/v1/
```
**uname -a:** `Linux dhcp-136-140.nay.redhat.com 4.1.6-200.fc22.x86_64 #1 SMP Mon Aug 17 19:54:31 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux`

**List the steps to reproduce the issue:**
1. Download fedora 22 image use 'docker pull fedora:22'
2. Create a container with F22 image and use 'uname -a' to check linux version. It will show linux version is f22.

Download fedora 21 image use 'docker pull fedora:21'
Create F21 container and use 'uname -a', result as below:
```
[wangdong@dhcp-136-140 tools-automation]$ sudo docker run -it --name dowang_test21 fedora:21 /bin/bash
bash-4.3# uname -a
Linux 5851876fbb32 4.1.6-200.fc22.x86_64 #1 SMP Mon Aug 17 19:54:31 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
bash-4.3# dnf
bash: dnf: command not found
bash-4.3# exit
```

**Describe the results you received:**
Why my container use a fedora 21 image, but it show f22 in uname -a .

**Describe the results you expected:**
In fedora 21 container it should show f21 in uname -a

**Provide additional info you think is important:**
In my folks computer, he download f21 image first than download f22, in his f22 container 'uname -a' always show as f21

----------END REPORT ---------

**Comment by unclejack** [comment]
@What you see in `uname -a` comes from the host, not from the container. You're seeing `4.1.6-200.fc22.x86_64` in the container because your host is running Fedora 22 and that's your host's kernel.

This isn't a general support forum. This is an issue tracker for Docker bugs, features and pull requests. Please take this into account when opening an issue.

This isn't a Docker bug, this is a question which is more appropriate for
1) the forums: https://forums.docker.com/
2) the docker-user mailing list https://groups.google.com/forum/#!forum/docker-user
3) the #docker channel on Freenode.

1) and 2) should be better for longer conversations.

Since this isn't a bug, I'm going to close this now. Please feel free to comment.
